canvas_modules/harness/src/client/components/sidepanel.jsx

```diff
@@ -60,14 +57,19 @@ export default class SidePanel extends React.Component {
 				break;
 			default:
 			}
+			return (
+				<aside aria-label="Right Side Panel" role="complementary">
+					<div className={"harness-app-sidepanel open"}>
+						{view}
+					</div>
+				</aside>
+			);
 		}
 
+		// Return just a div when flyout is closed. The div will allow
```

Comment:
I noticed the test harness side panel now appears right away and only common-canvas animates to size. Is this the expected behavior with this change?

Since this is only the test harness, either way doesnt matter

Reply:
Hi Carita - Good catch. I had it working an must have messed something up. I am trying to get the test harness panels to not be involved in voice over when the panel is closed. Anyway, I think I fixed it can you review again please?